includes/admin/class-evf-admin-import-export.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * EverestForms Import Export Class
+ *
+ * @package EverestForms\Admin
+ * @since   1.5.11
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * EVF_Admin_Forms class.
+ */
+class EVF_Admin_Import_Export {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		add_action( 'admin_init', array( $this, 'export_json' ) );
+	}
+
+	/**
+	 * Exports form data along with settings in JSON format.
+	 *
+	 * @return void
+	 */
+	public function export_json() {
+		global $wpdb;
```

Comment:
Empty line needed after global var declaration.

Reply:
The spacing has been added.